packages/suite-data/files/translations/en.json

```diff
@@ -353,11 +353,11 @@
   "TR_BACKUP_SUBHEADING_1": "A wallet backup is a series of randomly generated words created by your Trezor. It’s important to write down your wallet backup and keep it safe, as it's the only way to recover and access your funds.",
   "TR_BACKUP_SUCCESSFUL": "Backup successful",
   "TR_BALANCE": "Balance",
+  "TR_CHECK_BACKUP": "Check backup",
   "TR_BASIC_RECOVERY": "Standard recovery",
   "TR_BASIC_RECOVERY_OPTION": "Enter your wallet backup word by word on your computer.",
   "TR_BCH_ADDRESS_INFO": "Bitcoin Cash changed the address format to cashaddr. Find more info about how to convert your address on our blog. {TR_LEARN_MORE}",
-  "TR_BEFORE_ANY_FURTHER_ACTIONS": "Although unlikely, you may need to access your wallet backup in case of a firmware update issue.",
-  "TR_BIP_SIG_FORMAT": "Trezor",
+  "TR_BEFORE_ANY_FURTHER_ACTIONS": "Although unlikely, you may need to access your wallet backup in the event of a firmware update issue. If you’re uncertain about the accuracy of your backup, you can verify it before proceeding with the installation.",
```

Comment:
the copy seems too long. I would not read it. How about using yellow warning banner